test/e2e/map-modify-actions.test.js

```diff
@@ -0,0 +1,138 @@
+const {assert, getDriver} = require('vl-ui-core').Test.Setup;
+const VlMapModifyActionsPage = require('./pages/vl-map-modify-actions.page');
+
+describe('vl-map-modify-action', async () => {
+  let vlMapPage;
+  const willebroek = [150390.63, 193371.12];
+
+  before(() => {
+    vlMapPage = new VlMapModifyActionsPage(getDriver());
+    return vlMapPage.load();
+  });
+
+  it('als gebruiker kan ik een polygoon aanpassen op een kaart', async () => {
+    const action = await vlMapPage.getModifyPolygonAction();
+
+    const layer = await action.getLayer();
+    await getDriver().wait(async () => (await layer.getNumberOfFeatures()) === 1);
+
+    const polygonFeature = await layer.getFeature(1);
+    const polygonCoordinates = polygonFeature.geometry.coordinates;
+    assert.lengthOf(polygonCoordinates, 1);
+
+    const points = polygonCoordinates[0];
+    assert.lengthOf(points, 4);
+
+    const gent = points[0];
+    assert.lengthOf(gent, 2);
+
+    const mechelen = points[1];
+    assert.lengthOf(mechelen, 2);
+
+    const antwerpen = points[2];
+    assert.lengthOf(antwerpen, 2);
+
+    const leuven = points[3];
+    assert.lengthOf(leuven, 2);
+
```

Comment:
zijde hier nu gewoon aan het asserten dat elke coordinaat uit nen X en nen Y (array van 2) bestaat? lijkt mij wat overkill?

Reply:
ja klopt, zag dat in sommige testen het aantal features als pre conditie getest wordt. Voor mij hoeft het op zich niet maar het geeft wat extra controle op je test setup. Omdat je demo en test case op elkaar steunen vind ik het nu ook niet super erg.

Reply:
Er is nog een verschil tussen 

- testen hoeveel features er zijn
- testen uit hoeveel punten een feature bestaat
- testen dat elke coordinaat van een feature uit 2 bestaat (X - Y)

Dat laatste lijkt me een given dat je zeker niet telkens opnieuw moet doen.

Reply:
ok, haal ze weg en pas het overal aan